python/cudf/cudf/core/series.py

```diff
@@ -31,6 +31,7 @@
 from cudf.core.column.lists import ListMethods
 from cudf.core.column.string import StringMethods
 from cudf.core.column_accessor import ColumnAccessor
+from cudf.core.dataframe import DataFrame
```

Comment:
Using `@copy_docstring(cudf.DataFrame.describe)` is leading to a circular dependency issue I suppose:
```python
>>> import cudf
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/nvme/0/pgali/envs/cudfdev/lib/python3.7/site-packages/cudf/__init__.py", line 11, in <module>
    from cudf import core, datasets, testing
  File "/nvme/0/pgali/envs/cudfdev/lib/python3.7/site-packages/cudf/core/__init__.py", line 24, in <module>
    from cudf.core.series import Series
  File "/nvme/0/pgali/envs/cudfdev/lib/python3.7/site-packages/cudf/core/series.py", line 53, in <module>
    class Series(Frame, Serializable):
  File "/nvme/0/pgali/envs/cudfdev/lib/python3.7/site-packages/cudf/core/series.py", line 3838, in Series
    @copy_docstring(cudf.DataFrame.describe)
AttributeError: module 'cudf' has no attribute 'DataFrame'
```

Instead defined a common docstring in docutils and using that as a decorator for both APIs to avoid this circular dependency.